autokeras/blocks/preprocessing.py

```diff
@@ -211,9 +211,9 @@ def build(self, hp, inputs=None):
             flip_mode = ""
         elif horizontal_flip and vertical_flip:
             flip_mode = "horizontal_and_vertical"
-        elif horizontal_flip and not vertical_flip:
+        elif horizontal_flip:
             flip_mode = "horizontal"
-        elif not horizontal_flip and vertical_flip:
+        else:
```

Comment:
The redundancy here is for readability.
I prefer to leave it as it was.

Reply:
@haifeng-jin I understand that, then better close the PR

Reply:
@haifeng-jin sorry misunderstanding, you mean the preprocessing should stay as it is and then you are fine?